src/app/CASEClient.cpp

```diff
@@ -19,7 +19,10 @@
 
 namespace chip {
 
-CASEClient::CASEClient(const CASEClientInitParams & params) : mInitParams(params) {}
+CASEClient::CASEClient(const CASEClientInitParams & params) : mInitParams(params)
+{
+    mCASESession.Init(params.sessionManager);
```

Comment:
Is the RAM cost worth the (mixed, apparently) flash savings here?

Reply:
Yes, Judging from bloat result, this doesn't work well.